blog/20111224new-wiki-and-new-distributions-supported.md

```diff
@@ -8,8 +8,8 @@
 .. author: Steve Zesch
 -->
 
-We've recently switched from a wiki at [GitHub](http://github.com/) to a our own
-available from [wiki.mate-desktop.org](http://wiki.mate-desktop.org). Please
+We've recently switched from a wiki at [GitHub](https://github.com/) to a our own
+available from [wiki.mate-desktop.org](https://wiki.mate-desktop.org). Please
```

Comment:
http://wiki.mate-desktop.org redirects to https://wiki.mate-desktop.org and it doesn't work, dead link

Reply:
Of course, because i need to enable `http://wiki.mate-desktop.org` as custom domain name for new wiki page at https://github.com/mate-desktop/mate-wiki/settings
`https://mate-desktop.github.io/mate-wiki/#!index.md` to `http://wiki.mate-desktop.org`

I hope this will work after this PR.

Reply:
ok, I will review all the links, it will take me some time... I will tell you when I finish.

Reply:
Thank you